lib/scrabble/scrabble.rb

```diff
@@ -0,0 +1,121 @@
+module Scrabble
+  class Scrabble
+    ## -------------------------------------------------------------------------
+    # CONSTANTS
+
+
+    LETTER_SCORES = [
+      { points: 1,  letters: %w{a e i o u l n r s t} },
+      { points: 2,  letters: %w{d g} },
+      { points: 3,  letters: %w{b c m p} },
+      { points: 4,  letters: %w{f h v w y} },
+      { points: 5,  letters: %w{k} },
+      { points: 8,  letters: %w{j x} },
+      { points: 10, letters: %w{q z} }
+    ]
+
+    EMPTY_WORD_ERROR = "Please enter a word before hitting the Score Word button."
```

Comment:
Abstracting out error messages is very important, especially if you ever need to consider internationalization (i.e. translating your app into another language), so I'm super impressed you decided to do that.